### PR TITLE
Add Roku Nerdpack setup to Quickstart

### PR DESCRIPTION
**Pull request by cjmccarthy**
Adding a link to marketplace for our Roku Nerdpack

### CODE REVIEW

File: install/third-party/roku-http-analytics/install.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: |
+  An HTTP and HTTP Errors analytics experience for Roku to identify and fix networking issues
+target:
+  type: unknown
+  destination: unknown
+
+install:
+  mode: link
+  destination:
+    url: https://one.newrelic.com/marketplace?state=cc2a568b-0d2b-8498-aec4-c4230bf4d2a0
```
[the remark this answers is not in the document]
Comment (cjmccarthy) [reply install/third-party/roku-http-analytics/install.yml:13]: I'm not sure I understand how it works, I'll have to gather more information and come back to this

[the remark this answers is not in the document]
Comment (cjmccarthy) [reply install/third-party/roku-http-analytics/install.yml]: Ok, updated!

[the remark this answers is not in the document]
Comment (aswanson-nr) [reply install/third-party/roku-http-analytics/install.yml]: Hey @cjmccarthy, sorry if I made things a bit more complicated. Just to be sure this is what we want, what exactly should happen when a user installs this quickstart?
Are they thrown into a setup nerdlet, sent to a docs page, or something else?
It's worth noting that the nerdlet must be available to all NR1 accounts to be used in this fashion.

[the remark this answers is not in the document]
Comment (cjmccarthy) [reply install/third-party/roku-http-analytics/install.yml]: The initial intent was to open the marketplace install screen for this nerdpack (which contains two nerdlets, one of which is now listed here directly). My understanding (correct if I'm wrong @amit-y ) is that this change will simply open the installation for the one specified nerdlet. If it works as intended I'll fast-follow another PR with a second install plan for the next nerdlet

[the remark this answers is not in the document]
Comment (cjmccarthy) [reply install/third-party/roku-http-analytics/install.yml]: @aswanson-nr does that change make sense? If so, does this PR look good? Thanks!

[the remark this answers is not in the document]
Comment (josephgregoryii) [reply install/third-party/roku-http-analytics/install.yml]: I checked to make sure the install works - and it does! thanks for this update 😄

[the remark this answers is not in the document]
Comment (aswanson-nr) [reply install/third-party/roku-http-analytics/install.yml]: @josephgregoryii, thank you for checking on this! Looks like things are in order. Let me get this resolved so we can move forward.

**Review by aswanson-nr — APPROVED** [review]

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**